cluster/gce/gci/configure-helper-common.sh

```diff
@@ -2353,7 +2353,7 @@ function start-kube-controller-manager {
     RUN_CONTROLLERS="serviceaccount,serviceaccount-token,nodelifecycle,nodeipam,ttl,csrsigning,csrapproving,csrcleaner"
   fi
   if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "tp" ]]; then
-    RUN_CONTROLLERS="*,-nodeipam,-nodelifecycle,-mizar-controllers,-network,-ttl"
```

Comment:
is there specific reason to remove "-mizar-controllers, -network" here?

Reply:
Mizar controller needs to run in the TP doesn't it?

Reply:
This is aligned with local dev config.